Content.Server/Shuttles/Systems/EmergencyShuttleSystem.cs

```diff
@@ -77,9 +80,15 @@ public override void Initialize()
         SubscribeLocalEvent<StationCentcommComponent, ComponentShutdown>(OnCentcommShutdown);
         SubscribeLocalEvent<StationCentcommComponent, ComponentInit>(OnCentcommInit);
         SubscribeNetworkEvent<EmergencyShuttleRequestPositionMessage>(OnShuttleRequestPosition);
+        SubscribeLocalEvent<RoundEndTextAppendEvent>(OnRoundEnded);
         InitializeEmergencyConsole();
     }
 
+    private void OnRoundEnded(RoundEndTextAppendEvent ev)
+    {
+        DockTime = null;
+    }
+
```

Comment:
Use https://github.com/space-wizards/space-station-14/pull/19471 and device networking.

Reply:
> So i just wait merge shuttle timers and after merge get dock time from timers?

the docktime is stored in the ArrivalsShuttleComponent (NextTransfer)

Reply:
> > So i just wait merge shuttle timers and after merge get dock time from timers?
> 
> the docktime is stored in the ArrivalsShuttleComponent (NextTransfer)

But i need Evacuation shutte, not arriaval

Reply:
> > > So i just wait merge shuttle timers and after merge get dock time from timers?
> > 
> > 
> > the docktime is stored in the ArrivalsShuttleComponent (NextTransfer)
> 
> But i need Evacuation shutte, not arrival

oh, sorry i misread from the title. that's not stored internally but grabbed from a cvar when the shuttle is called and spawns a timer/countdown token for recalls. if we're at a point where 3 things are independently tracking the evac shuttle data then we should refactor the emergency shuttle and console system to store this in a component; the console system is jank as hell.

though this won't be as necessary once the hallway screens are merged

Reply:
> Use #19471 and device networking.

i dont understand how use networking in this case.

Reply:
> Use #19471 and device networking.

I remove DockTime = null and use other solve. Please check this.